src/gettingstarted/own-code/free-trial.md

```diff
@@ -0,0 +1,21 @@
+# Import your own code
+
+## Open a free trial account
```

Comment:
If this page is going to be repeated, is there a way to make it an include for single-sourcing?  (Maybe not in GitBook; if not, meh.)

Reply:
I was able to get this working on an earlier version, so I'll see if I can find that.